.github/workflows/formatting.yml

```diff
@@ -0,0 +1,59 @@
+name: Formatting
+on:
+  push:
+    branches:
+      - develop
+  pull_request:
+    branches:
+      - develop
+
+jobs:
+  pre-commit:
+    if: github.event.pull_request.draft != true && github.event.pull_request.head.repo.full_name == github.repository
```

Comment:
This condition could get applied at the step level for staging changed files and creating the PR instead of the entire job.

Reply:
Can you check the change, it doesn't seem to skip it in the right circumstances, not sure what I am doing wrong

Reply:
Skipping the PR creation steps is the expected behavior right now?

Reply:
right since it is a draft PR